screenpy_playwright/target.py

```diff
@@ -87,11 +91,15 @@ def __repr__(self) -> str:
             comma = ", "
         return f"{self.name}{left_paren}{args}{comma}{kwargs}{right_paren}"
 
+    def __repr__(self) -> str:
+        """Return the Target for representation."""
+        return repr(self.target)
```

Comment:
This change handles the Target being correctly logged. Before, for a Target like:
```python
t = Target.the("Ossuary door").located_by("graveyard > building door").first
```

Logging would be like:
> Tor knocks on the first

rather than what it is now:
> Tor knocks on the Ossuary door